libs/libarchive/Makefile

```diff
@@ -104,6 +104,14 @@ else
 	CMAKE_OPTIONS += -DENABLE_OPENSSL=ON
 endif
 
+EXTRA_CFLAGS += "-I$(PKG_BUILD_DIR)/extra-includes"
```

Comment:
can't do

```
EXTRA_CFLAGS += "-I$(STAGING_DIR_HOST)/include/ext2fs"
```
?

Reply:
IMO leaking host path into target's cross compile environment is probably not a good idea.